packages/scripts/package.json

```
@@ -66,6 +66,7 @@
 		"mini-css-extract-plugin": "^1.3.9",
 		"minimist": "^1.2.0",
 		"npm-package-json-lint": "^5.0.0",
+		"optimize-css-assets-webpack-plugin": "6.0.1",
```

```suggestion
		"optimize-css-assets-webpack-plugin": "^6.0.1",
```
For consistency with the other dependencies, we should probably use the `^` modifier.

---

Agreed. Thanks, PR updated.